### PR TITLE
Remove unsupported OS

### PR DESCRIPTION
**Pull request by Valantin**
#### Pull Request (PR) description
Remove unsupported os

#### This Pull Request (PR) fixes the following issues
#954

### CODE REVIEW

**Comment by bastelfreak** [comment]
Can you please rebase against our master branch to get rid of the merge commit?

File: manifests/repo.pp [hunk 1]
```diff
@@ -18,19 +18,9 @@
...
-        $majorrelease = '6'
-      }
-      default: {
-        $majorrelease = $facts['os']['release']['major']
-      }
-    }
     case $facts['os']['family'] {
       'RedHat': {
+        $majorrelease = $facts['os']['release']['major']
         if (versioncmp(fact('os.release.major'), '7') >= 0 and $zabbix_version == '7.0') {
```
Comment (bastelfreak) [inline manifests/repo.pp:24]: you can use `$majorrelease` here
```suggestion
        if (versioncmp($majorrelease, '7') >= 0 and $zabbix_version == '7.0') {
```

File: metadata.json [hunk 2]
```diff
@@ -79,18 +72,10 @@
...
-      "operatingsystem": "Scientific",
-      "operatingsystemrelease": [
-        "7",
-        "8"
-      ]
-    },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
         "7",
```
Comment (bastelfreak) [inline metadata.json:78]: CentOS 7 is also EoL: https://voxpupuli.org/blog/2024/05/08/CentOS-7-CentOS-Stream-8-deprecation/

Comment (Valantin) [reply metadata.json:78]: Yes I'v planned to remove EL7 code and OS in a separe PR, need to remove a lot of unused logic from manifest and spec

**Pull request merged**